feature/2068 - Add KMHT

jlambert134 opened the pull request:
Resolves #2068 

The purpose of this pull request is to add the progress from KMHT to the development base

jessie846 committed:
added KMHT

jlambert134 committed:
Update kmht.json

jlambert134 committed:
Update kmht.json

jlambert134 committed:
Update kmht.json

erikquinn committed:
feature/2068 - Clean up KMHT slightly

jlambert134 committed:
Update kmht.json

jlambert134 committed:
Update kmht.json

jlambert134 committed:
Update kmht.json

jlambert134 committed:
Update kmht.json

jlambert134 committed:
Update kmht.json

jlambert134 committed:
Update kmht.json

jlambert134 committed:
Create kmht.geojson

jlambert134 committed:
Update kmht.json

jlambert134 committed:
Update kmht.json

jlambert134 committed:
Update kmht.json

jlambert134 committed:
Update kmht.json

jlambert134 committed:
Update kmht.json

jlambert134 committed:
Update kmht.json

jlambert134 committed:
Update kmht.json

jlambert134 committed:
Create kmht.md

jlambert134 committed:
Fixed and added documentation

jlambert134 committed:
Update kmht.md

jlambert134 committed:
Update kmht.md

jlambert134 committed:
Update kmht.md

jlambert134 committed:
Update kmht.json

jlambert134 committed:
Update kmht.json

jlambert134 committed:
Update kmht.json

Fechulo commented on assets/airports/kmht.json:
```diff
@@ -0,0 +1,976 @@
+                "KMHT17": [["TYLIR", "A30+"]],
+                "KMHT35": [["TYLIR", "A30+"]]
+            },
+            "body": ["TYLIR", "PPORT", ["SHOEL", "A50+"], ["RBELA", "A68+"], ["COTEE", "A93+"]],
```
NELIE and CCC transitions skip over COTEE, so it should be moved to the BAF exitPoint definition.

There are no altitude restrictions on this SID other than at TYLIR. The altitude restrictions are MSAs so shouldn't be defined as part of the SID.

jlambert134 replied:
Thank you, I really didnt read that part of the chart.

jlambert134 replied:
✔️

Fechulo commented on assets/airports/kmht.json:
```diff
@@ -0,0 +1,976 @@
+            "body": ["TYLIR", "PPORT", ["SHOEL", "A50+"], ["RBELA", "A68+"], ["COTEE", "A93+"]],
+            "exitPoints": {
+                "BAF":[["BAF", "A168"]],
+                "CCC":["JJIMY", ["YANTC", "A163+"], ["PPORT", "A170"]],
```
Should be CCC, not PPORT

jlambert134 replied:
THANK YOU, during my playtest aircraft kept coming back 😆

jlambert134 replied:
✔️

Fechulo commented on assets/airports/kmht.json:
```diff
@@ -0,0 +1,976 @@
+        "icao": "ROZZE2",
+        "name": "Rozze Two",
+        "entryPoints": {
+            "MUCOW": [["MUCOW", "A190-|S280"], ["FFATY", "A200-|A160+"], ["CRIBB", "A160-|A130+"], ["MONRC", "A90+"]],
```
MUCOW is at or _above_ FL190, so it should read A190+.

Altitude restrictions should be written with the lower first and the higher last. so FFATY should be A160+|A200-

jlambert134 replied:
✔️

Fechulo commented on assets/airports/kmht.json:
```diff
@@ -0,0 +1,976 @@
+        },
+        "body": ["ROZZE"],
+        "rwy": {
+            "KMHT6" : [["PNARD", "A90-|A70+|S230"], ["TEETO", "A50"]],
```
TEETO and JSTNN should be S210

jlambert134 replied:
✔️

Fechulo commented on assets/airports/airportLoadList.json:
```diff
@@ -283,6 +283,12 @@
     },
+    {
+        "icao": "kmht",
+        "level": "easy",
```
With the current traffic levels, it falls under 'medium'. Between 20 and 40 acph is 'easy'

jlambert134 replied:
✔️

Fechulo commented on documentation/airport-guides/kmht.md:
```diff
@@ -0,0 +1,55 @@
+# KMHT - Manchester-Boston Regional Airport (A90 FITZY Sector)
+
+## About MHT
+Manchester-Boston Regional Airport (IATA: MHT, ICAO: KMHT) is a civil airport located right in the heart of Manchester, NH, and about 1 hour away from the major New England city of Boston, MA. MHT iis the primary airport for aircraft arriving into the state of New Hampshire. Although MHT does not hold hubs for any airlines, MHT is a major flying destination for Freight Runner Airlines and Wiggins ...
```
typo "iis"

jlambert134 replied:
✔️

jlambert134 committed:
Major typo/SID & STAR errors fixed.

All things needed to be fixed for the review, were fixed. a few other small errors were fixed.

Fechulo commented on assets/airports/kmht.json:
```diff
@@ -174,10 +174,10 @@
                 "KMHT35": [["TYLIR", "A30+"]]
             },
-            "body": ["TYLIR", "PPORT", ["SHOEL", "A50+"], ["RBELA", "A68+"], ["COTEE", "A93+"]],
+            "body": ["TYLIR", "PPORT", ["SHOEL", "A50+"], ["RBELA", "A68+"]],
```
- Remove TYLIR from body, it is already defined in the runways (or remove them from the runways and add the altitude restriction in the body)
- Remove altitude restrictions at SHOEL and RBELA

Fechulo commented on assets/airports/kmht.json:
```diff
@@ -174,10 +174,10 @@
             "exitPoints": {
-                "BAF":[["BAF", "A168"]],
-                "CCC":["JJIMY", ["YANTC", "A163+"], ["PPORT", "A170"]],
+                "BAF":[["COTEE", "A93+"], ["BAF", "A168"]],
```
Remove altitude restrictions at COTEE and BAF

Fechulo commented on assets/airports/kmht.json:
```diff
@@ -174,10 +174,10 @@
-                "BAF":[["BAF", "A168"]],
-                "CCC":["JJIMY", ["YANTC", "A163+"], ["PPORT", "A170"]],
+                "BAF":[["COTEE", "A93+"], ["BAF", "A168"]],
+                "CCC":["JJIMY", ["YANTC", "A163+"], ["CCC", "A170"]],
```
Remove altitude restrictions at YANTC and CCC

Fechulo commented on assets/airports/kmht.json:
```diff
@@ -174,10 +174,10 @@
-                "CCC":["JJIMY", ["YANTC", "A163+"], ["PPORT", "A170"]],
+                "BAF":[["COTEE", "A93+"], ["BAF", "A168"]],
+                "CCC":["JJIMY", ["YANTC", "A163+"], ["CCC", "A170"]],
                 "NELIE":["JJIMY", ["NELIE", "A170"]]
```
Remove altitude restrictions at NELIE

jlambert134 committed:
fixed changes requested

jlambert134 committed:
Update kmht.json

Fechulo reviewed (changes requested):
Nearly there now I think!
Having play tested it for a bit, I think it'd be good to add some more variety in the arrival traffic. Obviously there's only one STAR, but looking at FlightAware it looks like Wiggins do a number of flights from nearby airports. I think adding a few of their flights coming from other directions would make this much more fun to play. Adding some other light aircraft in the mix would be great too
